dashboard/db/migrate/20160630000000_add_pd_enrollment_survey_fields.rb

```diff
@@ -0,0 +1,9 @@
+class AddPdEnrollmentSurveyFields < ActiveRecord::Migration
+  def change
+    add_column :pd_enrollments, :survey_sent_at, :datetime
+    add_column :pd_enrollments, :survey_id, :integer
```

Comment:
The survey id is for a completed survey, so it makes sense to go after. The survey itself it keyed on the enrollment code so we already have that data. I'll add comments and change `survey_id` to `completed_survey_id`.